hall-of-fame-service/src/main/java/hcmus/alumni/halloffame/controller/HallOfFameServiceController.java

```diff
@@ -0,0 +1,296 @@
+		}
+
+		return ResponseEntity.status(HttpStatus.OK).body(result);
+	}
+
+//
+//	@GetMapping("")
+//	public ResponseEntity<HashMap<String, Object>> getHallOfFame(
+//			@RequestParam(value = "offset", required = false, defaultValue = "0") int offset,
+//			@RequestParam(value = "limit", required = false, defaultValue = "10") int limit,
+//			@RequestParam(value = "keyword", required = false, defaultValue = "") String keyword,
+//			@RequestParam(value = "createAtOrder", required = false, defaultValue = "desc") String createAtOrder) {
+//		// Initiate
+//		CriteriaBuilder cb = em.getCriteriaBuilder();
+//		CriteriaQuery<HallOfFameDto> cq = cb.createQuery(HallOfFameDto.class);
+//		// From
+//		Root<HallOfFameModel> root = cq.from(HallOfFameModel.class);
+//
+//		// Join
+//		Join<HallOfFameModel, StatusPostModel> statusJoin = root.join("status", JoinType.INNER);
+//		
+//		// Select
+//		Selection<String> idSelection = root.get("id");
+//		Selection<String> titleSelection = root.get("title");
+//		Selection<String> contentSelection = root.get("content");
+//		Selection<String> thumbnailSelection = root.get("thumbnail");
+//		Selection<Integer> viewsSelection = root.get("views");
+//		Selection<String> facultySelection = root.get("faculty");
+//		Selection<Integer> beginningYearSelection = root.get("beginningYear");
+//		cq.multiselect(idSelection, titleSelection, contentSelection, thumbnailSelection, viewsSelection, facultySelection, beginningYearSelection);
+//		
+//		// Where
+//		Predicate titlePredicate = cb.like(root.get("title"), "%" + keyword + "%");
+//		Predicate statusPredicate = cb.equal(statusJoin.get("name"), "Bình thường");
+//		cq.where(titlePredicate, statusPredicate);
+//		
+//		// Order by
+//		List<Order> orderList = new ArrayList<Order>();
+//		if (createAtOrder.equals("asc")) {
+//			orderList.add(cb.asc(root.get("createAt")));
+//		} else if (createAtOrder.equals("desc")) {
+//			orderList.add(cb.desc(root.get("createAt")));
+//		}
+//		cq.orderBy(orderList);
+//		
+//		HashMap<String, Object> result = new HashMap<String, Object>();
+//		TypedQuery<HallOfFameDto> typedQuery = em.createQuery(cq);
+//		result.put("itemNumber", typedQuery.getResultList().size());
+//		typedQuery.setFirstResult(offset);
+//		typedQuery.setMaxResults(limit);
+//		result.put("items", typedQuery.getResultList());
+//		
+//		return ResponseEntity.status(HttpStatus.OK).body(result);
+//	}
+//
+	@PreAuthorize("hasAnyAuthority('Admin')")
+	@PostMapping("")
+	public ResponseEntity<String> createHallOfFame(@RequestHeader("userId") String creator,
+			@RequestParam(value = "title") String title, @RequestParam(value = "content") String content,
+			@RequestParam(value = "thumbnail") MultipartFile thumbnail,
+			@RequestParam(value = "faculty") Integer faculty, @RequestParam(value = "emailOfUser") String emailOfUser,
+			@RequestParam(value = "beginningYear") Integer beginningYear) {
```

Comment:
```suggestion
			@RequestParam(value = "beginningYear") Integer beginningYear), @RequestParam(value = "scheduledTime", required = false) Long scheduledTimeMili) {
```
thấy có làm lên lịch mà ở đây thiếu cái schudule